content/en/blog/2023/http-conventions-declared-stable.md

```diff
@@ -0,0 +1,264 @@
+
+We also want to share some of the reasons behind these changes, as we think that the stable version brings many
+significant improvements which will make the pain of this upgrade worth it in the long run.
+
+## Aligning with other open standards
+
+Earlier this year we announced [the merging of Elastic Common Schema into OpenTelemetry Semantic Conventions]([https://opentelemetry.io/blog/2023/ecs-otel-semconv-convergence/]).
+
+This merger brought several direct benefits to the HTTP semantic conventions.
+
+One was the transition from `net.host.*` and `net.peer.*`
+to `client.*` and `server.*`, which has a number of benefits:
+* works better for logs (where there's no span kind)
+* improves correlation across client and server telemetry (e.g. since you can join on `server.address` instead of joining where
+`net.peer.addr` == `net.host.addr`)
+* is generally more intuitive compared to `net.host.*` and `net.peer.*`
+* left open the `network.peer.*` and `network.local.*` namespace for low-level network instrumentation where peer and local concepts are more natural
+
+Another was the `url.*` namespace, which allows these attributes to be reused in non-HTTP semantic conventions
+in the future.
+
+And another was improved consistency of namespaces,
+e.g. `http.request.method` instead of `http.method`, and `http.response.status_code` instead of `http.status_code`,
+which match the other `http.request.*` and `http.response.*` attributes.
+
+Better alignment with Prometheus by adopting seconds as the standard unit for metrics.
+
+## Telemetry has a cost
+
+Just because we can capture something, doesn't mean we should capture it by default.
+
+Capturing, processing, and storing telemetry has a cost, and we should consider this when deciding which attributes
+are recommended vs opt-in.
+
+## Default values can be problematic
+
+Even though default values can help with cost, in some cases they can make it impossible for consumers of that
+telemetry to tell the difference between an uncaptured value and the default value.
+
+## Metric cardinality
+
+For example, in the previous version, `http.method` could be spammed by an attacker, causing an explosion in that
+one dimension. With the OpenTelemetry Metric SDK's (configurable) cardinality cap, this doesn't cause a memory
+explosion, but it does cause a degradation in your observability of HTTP metrics since the cardinality cap is hit
+just with that one dimension.
+
+In the stable version, `http.request.method` is limited to a (configurable) set of known HTTP methods.
+
+When there is an unknown value, it is stored in `http.request.method_original`, and `http.request.method` is set to `_OTHER`.
+
+This allows the same definition of `http.request.method` to be used on both spans and metrics, which supports correlation between spans and metrics
+(and also other use cases like span-to-metric pipelines, and instrumentations which unify attribute capture across both
+spans and metrics).
+
+# Transition plan
```

Comment:
You can't have a level-1 heading in this page. You'll need to promote all headings by one level below this point, e.g.:

```suggestion
## Transition plan
```